chain/network/src/peer_manager.rs

```diff
@@ -787,6 +787,14 @@ impl PeerManagerActor {
     /// Route signed message to target peer.
     /// Return whether the message is sent or not.
     fn send_signed_message_to_peer(&mut self, ctx: &mut Context<Self>, msg: RoutedMessage) -> bool {
+        // Check if the message is for myself and don't try to send it in that case.
```

Comment:
why would this happen to begin with?

Reply:
This is always the case with `ForwardTx`.